playpen/ansible/roles/dev/tasks/main.yml

```diff
@@ -55,3 +55,5 @@
 
 - name: Install custom ~/.bashrc
   copy: src=bashrc dest=/home/{{ ansible_env.SUDO_USER }}/.bashrc
+
```

Comment:
Probably it's just me, but the yaml already does that, and this just seems redundant.

meh :)